head/simulator/main.cpp

```diff
@@ -82,8 +82,13 @@ static motor_interrupt_driver::MotorInterruptDriver sim_interrupt_left(
 
 static auto adc_comms = adc::SimADC{};
 
+auto at = presence_sensing_driver::AttachedTool{
```

Comment:
Can `AttachedTool` have a constructor that assigns everything to `UNDEFINED_TOOL` instead of duping this initialization?